account_financial_report_qweb/models/account.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# © 2011 Guewen Baconnier (Camptocamp)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).-
+from openerp import models, fields
+
+
+class AccountAccount(models.Model):
+    _inherit = 'account.account'
+
+    centralized = fields.Boolean(
+        'Centralized',
+        default=False,
```

Comment:
For a boolean field, it is by default False.  So, no need to explicitly define it.